src/governance-object.h

```diff
@@ -25,6 +25,7 @@ class CGovernanceVote;
 
 static const int MAX_GOVERNANCE_OBJECT_DATA_SIZE = 16 * 1024;
 static const int MIN_GOVERNANCE_PEER_PROTO_VERSION = 70204;
+static const int GOVERNANCE_FILTER_PROTO_VERSION = 70206;
```

Comment:
You'd look there because it's a governance parameter and (being familiar with the code) you'd know that governance parameters are found near the top of governance-object.h.  Maybe the best solution is to create a third header file for them but I think that can wait for the next release.